src/Microsoft.Health.SqlServer/Configs/SqlServerDataStoreConfiguration.cs

```diff
@@ -14,6 +14,16 @@ public class SqlServerDataStoreConfiguration
     /// </summary>
     public const string SectionName = "SqlServer";
 
+    /// <summary>
+    /// The max pool size name used in connection string.
+    /// </summary>
+    public const string MaxPoolSizeName = "Max Pool Size";
```

Comment:
We should move this and the limit to a separate SQLConstants class. Currently this class represents everything that ic onfigurable

Reply:
Done